packages/es-client/tests/test-execution-upsert.js

```diff
@@ -44,15 +45,17 @@ test.serial('upsertExecution writes new "running" execution', async (t) => {
     index: esIndex,
   });
 
+  delete testRecord.randomKey;
   const record = await esExecutionsClient.get(testRecord.arn);
   t.like(record, testRecord);
 });
 
-test.serial('upsertExecution writes new "completed" execution', async (t) => {
+test.serial('upsertExecution writes new "completed" execution with null fields omitted', async (t) => {
```

Comment:
We need a test that shows nullification happens on overwrite as well as new writes.